aes_test.go

```diff
@@ -0,0 +1,45 @@
+package hdfs
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestAesChunks(t *testing.T) {
+	originalText := []byte("some random plain text, nice to have it quite long")
+	key := []byte("0123456789abcdef")
+
+	// choose iv to hit counter overflow
+	iv := []byte("\x00\x00\x00\x00\x00\x00\x00\x00\xff\xff\xff\xff\xff\xff\xff\xf5")
+	enc := &TransparentEncryptionInfo{iv: iv, key: key}
+
+	// one step encrypt/decrypt
```

Comment:
If you look through the comments in the repo, you'll see I prefer full sentences with thorough explanations. Sorry for being picky. I can update some of them before merging.

Reply:
Rebased with better comments.